tdesign/src/Icon/IconPencil.tsx

```diff
@@ -0,0 +1,8 @@
+import { pencilIconURI } from "./cssSvgStrings";
+import BaseIcon, { IIconProps } from "./BaseIcon";
+
+const IconPencil = (props: IIconProps) => {
+  return <BaseIcon iconSlug={"pencilGray"} svgURI={pencilIconURI} {...props} />;
```

Comment:
Minor, but it always irks me :smile: 

```suggestion
  return <BaseIcon iconSlug="pencilGray" svgURI={pencilIconURI} {...props} />;
```

Reply:
Same thing applies here, this file is autogenerated, so not really worth changing just one of them

Reply:
Sure, I'll resolve the thread then